### PR TITLE
Add method popN() to Navigator.

### PR DESCRIPTION
**Pull request by yueshuaijie**
Compareing API of
`Navigator`(http://facebook.github.io/react-native/releases/next/docs/navigator.html)
and
`NavigatorIOS`(http://facebook.github.io/react-native/releases/next/docs/navigatorios.html),

we found `Navigator` object lacks of the method `popN()` which is sometimes useful. This PR add it.

### CODE REVIEW

File: Libraries/CustomComponents/Navigator/Navigator.js [hunk 1]
```diff
@@ -1072,14 +1072,17 @@ var Navigator = React.createClass({
...
-  _popN: function(n) {
-    if (n === 0) {
+  /**
+   * Go back N scenes at once. When N=1, behavior matches `pop()`.
+   * When N is invalid(negative or bigger than current routes count), do nothing.
+   * @param {number} n The number of scenes to pop. Should be an integer.
+   */
+  popN: function(n) {
+    invariant(typeof n === 'number', 'Must supply a number to popN');
+    n = parseInt(n);
```
Comment (yueshuaijie) [inline Libraries/CustomComponents/Navigator/Navigator.js]: Fixed in 99673f4e8d3d429adbe89bca22370aed0cfefc87.

**Comment by mkonicek** [comment]
cc @ericvicenti @hedgerwang

**Comment by ericvicenti** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-8** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/844481042349541/int_phab) to review internal test results.

**Pull request closed without merging**